src/core/packages/ui-settings/server-internal/src/ui_settings_config.ts

```diff
@@ -9,19 +9,30 @@
 
 import { schema, TypeOf, offeringBasedSchema } from '@kbn/config-schema';
 import type { ServiceConfigDescriptor } from '@kbn/core-base-server-internal';
+import { DEFAULT_THEME_NAME } from '@kbn/core-ui-settings-common';
 import { ConfigDeprecationProvider } from '@kbn/config';
 
 const deprecations: ConfigDeprecationProvider = ({ unused, renameFromRoot }) => [
   unused('enabled', { level: 'warning' }),
   renameFromRoot('server.defaultRoute', 'uiSettings.overrides.defaultRoute', { level: 'warning' }),
 ];
 
+export const defaultThemeSchema = schema.oneOf([
+  // TODO: Remove amsterdam theme
+  // https://github.com/elastic/eui-private/issues/170
+  schema.literal('amsterdam'),
+  schema.literal('borealis'),
```

Comment:
Should this be derived from `SUPPORTED_THEME_NAMES` keys?

Reply:
That's a great idea! But since the validation is typed, it shouldn't be an issue to keep it this way (as types will tell somewhere that these are not supported).